app/jobs/application_job.rb

```diff
@@ -1,4 +1,15 @@
 class ApplicationJob < ActiveJob::Base
+  include Foreman::Observable
+
+  # Foreman Subscription API delivery
+  after_perform do |job|
+    # Payload is a hash, the most important element is "arguments", to see more visit:
+    # https://github.com/rails/rails/blob/master/activejob/lib/active_job/core.rb (method serialize)
+    payload = { "job" => job.serialize }
+    hook_name = "#{payload['job']['job_class'].to_s.underscore}_performed"
+    trigger_hook hook_name, payload: payload
+  end
```

Comment:
Should this stay here now that we have `::Foreman::ObservableJob`?

Reply:
Yeah damn, forgot to remove these.